package.json

```diff
@@ -20,12 +20,13 @@
   },
   "dependencies": {
     "escape-string-regexp": "^2.0.0",
+    "execa": "^4.0.0",
     "is-plain-object": "^3.0.0"
   },
   "devDependencies": {
     "ava": "^3.0.0",
-    "execa": "^4.0.0",
     "nyc": "^15.0.0",
+    "typescript": "^3.7.5",
```

Comment:
We should make it a peer dependency as well.

Reply:
But people may have got different version installed... I think it should be up to them.

Reply:
We should set a minimum version based on the CLI we use.